### PR TITLE
Updated license text and headers, as recently done in qiskit-core

### PR DESCRIPTION
**Pull request by yaelbh**
Resolving Issue #5,

### CODE REVIEW

**Comment by yaelbh** [comment]
Note that the linter is currently failing because it can't find "test". This will be fixed with Issue #4.

File: examples/sympy_backends.py [hunk 1]
```diff
@@ -1,20 +1,9 @@
 # -*- coding: utf-8 -*-
-# pylint: disable=invalid-name,anomalous-backslash-in-string
 
-# Copyright 2017 IBM RESEARCH. All Rights Reserved.
```
Comment (abdonrd) [inline examples/sympy_backends.py]: We must keep the year of creation of the file.

Comment (yaelbh) [reply examples/sympy_backends.py]: Thanks @abdonrd for the review. Could you please elaborate on this comment. Should I modify 2017 to 2018 in all files? Or a subset of the files? Which ones?

Comment (abdonrd) [reply examples/sympy_backends.py]: @yaelbh the other files have 2018, so I assumed that they are from 2018.

And this quoted line is the only place where you have changed 2017 (so I assumed that it is from 2017) to 2018.

Comment (yaelbh) [reply examples/sympy_backends.py]: This is a new repository, created two weeks ago. Some of its files were originally created in qiskit-core in 2017, but moved to the new repository with some adjustments. Would you consider these files as created in 2017 or 2018?

Comment (abdonrd) [reply examples/sympy_backends.py]: Oh, well, in this particular case, I'm not 100% sure.
@jaygambetta thoughts?

Comment (yaelbh) [reply examples/sympy_backends.py]: Regarding the 2017 versus 2018 question, is there a conclusion?

Comment (abdonrd) [reply examples/sympy_backends.py]: @jaygambetta says it should be 2018.

**Pull request merged**